_commands/basics/wc.md

```diff
@@ -0,0 +1,84 @@
+---
+---
+
+wc
+-------
+
+Print the number of bytes, words, and lines in files
+
+~~~ bash
+a.txt: 
+I love EECS 398.
+What a great class. 
+Good.
+
+$ wc [filename] 
+$ wc a.txt
+3 9 44 a.txt
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+~~~ bash
+a.txt:
+What
+a great
+world we live
+in where Google has all the answers we need
+
+b.txt:
+wc is a great command.
+I love it.
+~~~
+
+
+#### `wc -c a.txt`
+~~~ bash
+$ wc -c a.txt
+71 a.txt
+~~~
+
+##### Break it down
+ * When the -c(equivalent to --bytes) flag is set, wc will output the number of bytes in the given file.
```

Comment:
Use Markdown's inline code syntax for these options (surround `-c` and `--bytes` in backticks `).

Please mirror this elsewhere in your documentation as well.